app/src/components/features/mocksV2/MockEditorIndex/Editor/index.tsx

```diff
@@ -388,47 +390,51 @@ const MockEditor: React.FC<Props> = ({
   };
 
   return (
-    <div className="overflow-hidden">
-      <MockEditorHeader
-        isNewMock={isNew}
-        mockType={mockType}
-        savingInProgress={savingInProgress}
-        handleClose={onClose}
-        handleSave={handleOnSave}
-        handleTest={handleTest}
-        setPassword={setPassword}
-        password={password}
-      />
-      <Col className="mock-editor-title-container">
-        <RQEditorTitle
-          name={name}
-          mode={isNew ? "create" : "edit"}
-          description={desc}
-          namePlaceholder={mockType === MockType.API ? "Mock name" : "File name"}
-          descriptionPlaceholder="Add your description here."
-          nameChangeCallback={onNameChange}
-          descriptionChangeCallback={onDescriptionChange}
-          tagText={fileType}
-          errors={errors}
+    <div className="mock-parent">
+      <BottomSheetProvider defaultPlacement={BottomSheetPlacement.RIGHT}>
+        <MockEditorHeader
+          isNewMock={isNew}
+          mockType={mockType}
+          savingInProgress={savingInProgress}
+          handleClose={onClose}
+          handleSave={handleOnSave}
+          handleTest={handleTest}
+          setPassword={setPassword}
+          password={password}
         />
-      </Col>
-      <div className="mock-editor-wrapper">
-        <div className="mock-editor-container">
-          <Row className="mock-editor-body">
-            {renderMetadataRow()}
-            {renderMockCodeEditor()}
-          </Row>
-        </div>
-      </div>
-      {!isNew ? (
-        <APIClient
-          request={apiRequest}
-          openInModal
-          modalTitle="Test mock endpoint"
-          isModalOpen={isTestModalOpen}
-          onModalClose={() => setIsTestModalOpen(false)}
-        />
-      ) : null}
+        <BottomSheetLayout layout="split" bottomSheet={<MockLogs mockId={id} />} minSize={0} hideBottomSheet={!id}>
```

Comment:
The BottomSheet layout has an enum named `SheetLayout`. update the layout prop to `SheetLayout.SPLIT`.

**Do this after merging master branch**